app/config/config.local.neon.example

```diff
@@ -26,6 +26,10 @@ parameters:
   monitor:
     address: wss://recodex.mff.cuni.cz:443/ws
 
+  localRegistration:
+    enabled: false  # set to true if you want to permit simple local registration
+    implicitGroupsIds: {}  # list of group IDs to which each newly registered user is added as a member
```

Comment:
I should be more specific... this options should not be here, only the `enabled` option